docs/fundamentals-and-concepts/stellar-data-structures/assets.mdx

```diff
@@ -9,6 +9,8 @@ Accounts on the Stellar network can be used to track, hold, and transfer any typ
 
 Assets on Stellar have two identifying characteristics: the asset code and the issuer. Since more than one organization can issue a credit representing the same asset, asset codes often overlap (for example, multiple companies offer a USD token on Stellar). Assets are uniquely identified by the combination of their asset code and issuer.
 
+Stellar assets differ from smart contract tokens, which you can read about in the [Tokens section](https://soroban.stellar.org/docs/reference/interfaces/token-interface) in the Soroban docs.
```

Comment:
Possible to add something here about how Stellar assets can be used in smart contracts, and how that's a recommended approach?  Right now, a reader might just jump to the token interface, when they probably want to keep reading to find the SAC.  "...however, it is possible, and even recommended in most cases, to wrap a Stellar asset using the Stellar Asset Contract for use in smart contracts rather than creating a Soroban token."